doc/classes/BakedLightmap.xml

```diff
@@ -18,12 +18,14 @@
 			<argument index="1" name="create_visual_debug" type="bool" default="false">
 			</argument>
 			<description>
+				Bakes the lightmaps within the currently-edited scene.
```

Comment:
There should not be an hyphen after an adverb ending in `-ly`, see https://www.dailywritingtips.com/adverbs-and-hyphens/
Same below.

Reply:
I've removed the hyphen on both of these.